### ISSUE TITLE
Replicating results of put-block-in-bowl-unseen-colors

### ISSUE THREAD

**Issue by mpiseno** [cliport/cliport#23]
Hi Mohit, I'm trying to replicate the results for the put-block-in-bowl-unseen-colors task in the paper for cliport (single) for 1000 training demonstrations, which is reported to have a test accuracy of 25%. However, I am getting a test accuracy of 47.2% and want to make sure what I'm doing is correct.

**Pipeline**
1. First I train the IL policy on put-block-in-bowl-seen-colors-train
2. Then I evaluate the all checkpoints of the policy on UNSEEN colors from a validation set. (i.e. I evaluate on a validation set of put-block-in-bowl-unseen-colors-val)
3. Then I run the best checkpoint on a test dataset of unseen colors. put-block-in-bowl-unseen-colors-test

**Questions**
1. Is this the correct training/evaluation pipeline? I believe so, from looking at the appendix of the paper.
2. Is there any way a discrepancy of test accuracies this large could have been introduced? Of course I expect our generated datasets to be different, but they should be generated from the same distribution, so I would not expect a difference of 47.2% vs 25%.

**Comment by MohitShridhar** [comment]
@mpiseno, this is strange.  

How big are your validation and test sets? Both 100 each? Have you tried visualizing the rollouts from evaluation? 

Also, can you post your train, val, and test configs or launch params.
